js/base/main.js

```diff
@@ -446,6 +460,15 @@ class RipeCommonsMainPlugin extends RipeCommonsPlugin {
                     this.$bus.trigger("format_change", format)
                 );
 
+                // registers a watch operation on all sdk options fields
+                // the data store so that the change is propagated to the ripe
+                // instance and then to the user interface

```

Comment:
Let's reword a bit: registers a watch operation on all options and updates the RIPE SDK accordingly.

The "user interface" part is misleading. You know that will happen, but not at this level of abstraction.